### PR TITLE
Sensu 0.19 prep

### PR DESCRIPTION
**Pull request by portertech**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
+## 0.19.0 - TBD
+
+### Features
+
+Redis Sensu transport, providing a built-in alternative to the default
```
Comment (calebhailey) [inline CHANGELOG.md]: s/providing/provides/

Comment (calebhailey) [inline CHANGELOG.md]: ...and drop the comma

Comment (portertech) [inline CHANGELOG.md]: I'll drop the unnecessary comma, but I think providing works/sounds better in this case, no?

Comment (portertech) [inline CHANGELOG.md]: Actually, going to keep the comma but drop the providing :P

**Pull request merged**